ibm/service/cis/resource_ibm_cis_alert_webhooks.go

```diff
@@ -0,0 +1,175 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
```

Comment:
rename file to ibm/service/cis/resource_ibm_cis_alert_webhook.go as this resource creates only one webhook